src/components/app.tsx

```diff
@@ -8,7 +8,7 @@ import * as SplitPane from 'react-split-pane';
 import { Dispatch } from 'redux';
 import { StateWithHistory } from 'redux-undo';
 import {Data, InlineData, isInlineData, isUrlData, UrlData} from 'vega-lite/build/src/data';
-import { datasetUrlLoad } from '../actions';
+import { datasetReceive, datasetUrlLoad } from '../actions';
```

Comment:
FYI, (a very very minor point) our codebase currently uses no space between {} in imports.  

(Maybe we can setup tslint and some autoformatter to fix this. I think we did set this in Vega-Lite, but I haven't ported the setup over.)